Makefile

```diff
@@ -54,11 +59,19 @@ format-check: ## Checks if format is correct
 .PHONY: install
 install: ## Update the package dependencies when new deps are added to dune-project
 	@opam install . --deps-only --with-test
-	@npm install
+	@npm install --force
```

Comment:
Why was `force` needed?

Reply:
Because testinglib and other deps depend on react 18, so npm complains. It is a decent workaround to ensure the installation succeeds

Reply:
Will write a comment

Reply:
would we still needed `--force` if you commit the lockfile from manual install and run `npm ci` here?